README.md

```diff
@@ -103,6 +103,10 @@ A `QBWC::Worker#requests` method cannot access values that are in-memory (global
 
 If requests are passed to `QBWC.add_job`, any `QBWC::Worker#requests` method will be ignored and will not be invoked during QuickBooks Web Connector sessions.
 
+### Referencing memory values when handling responses ###
+
+Similarly, a `QBWC::Worker#handle_response` method cannot access values that are in-memory at the time that `QBWC.add_job` is called; however, you can optionally pass arbitrary (serializable) data values to `QBWC.add_job`. This data will immediately be serialized and persisted by `QBWC.add_job`, then later deserialized and passed to `QBWC::Worker#handle_response` during a QuickBooks Web Connector session.
```

Comment:
-Maybe the title should be "Providing data to be made available when handling responses" or something like that. "memory values" doesn't really mean anything.
-"cannot access values" -> "cannot access variables"
-"arbitrary (serializable) data values" -> "a serializable value (for example, `String,` `Array`, or `Hash`)"
-In the last sentence, I don't think you need mention that it will be serialized and deserialized. Just saying it will be persisted and passed is good enough.